etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -125,3 +115,40 @@
       service:
         name: wazuh-dashboard
         state: restarted
+
+- hosts: wazuh-agent
+  become: yes
+  become_user: root
+  tasks:
+    - name: Check if custom SCA policies directory exists
+      stat:
+        path: "{{ local_custom_sca_policies_path }}"
+      register: custom_sca_policies_folder
+      delegate_to: localhost
+      become: no
```

Comment:
Would it not be clearer to become on each task that is necessary, rather than become on everything and invert it here?